provider/provider.go

```diff
@@ -212,7 +226,7 @@ func (p *dockerNativeProvider) Check(ctx context.Context, req *rpc.CheckRequest)
 	}
 
 	inputStruct, err := plugin.MarshalProperties(inputs, plugin.MarshalOptions{
-		KeepUnknowns: true,
+		KeepUnknowns: false,
```

Comment:
So this removes unknowns back in favor of empty values here, that's matching prior behavior I guess.